src/utils/collisions.ts

```diff
@@ -0,0 +1,44 @@
+import type { Collider, ColliderHandle, RigidBody, World } from '@dimforge/rapier3d-compat'
+import type { Group, Scene } from 'three'
+import type { Ref } from 'vue'
+
+interface CollisionSource {
+  collider: Collider
+  rigidBody: RigidBody | undefined
+};
+
+interface sourceTarget {
+  group: Group
+  rapierGroup: CollisionSource
```

Comment:
We can call `rapierGroup` -> `source` instead, it make more sense to me

Reply:
To be honest, I don't like rapierGroup... But I couldn't think of a better name

I don't like source because the event already uses the source name and a target, and it's too ambiguous.

Reply:
I see what you mean... And what about `context` or `ctx`?

Reply:
I like it, do you think "group" is a good name for the 3D object?

Reply:
Now that you mention it, maybe we can go with `object` instead

Reply:
Loved it. Commit is coming